job-service-acceptance-tests/pom.xml

```diff
@@ -455,6 +455,46 @@
                                 </log>
                             </run>
                         </image>
+    
+                        <!-- Run the Job Service Scheduled Executor -->
```

Comment:
Why is this being moved?

Reply:
The idea was to have it created before the jobservice container. Moved it back down